apps/console/src/features/roles/components/wizard-updated/role-basics.tsx

```diff
@@ -204,6 +225,15 @@ export const RoleBasics: FunctionComponent<RoleBasicProps> = (props: RoleBasicPr
      */
     const navigateToApplications = () => history.push(AppConstants.getPaths().get("APPLICATIONS"));
 
+    /**
+     * Navigate to the Applications Edit page.
+     */
+    const navigateToApplicationEdit = (appId: string) => 
+        history.push({
+            pathname: AppConstants.getPaths().get("APPLICATION_SIGN_IN_METHOD_EDIT")
+                .replace(":id", appId).replace(":tabName", `#tab=${ ROLES_TAB_INDEX }`)
+        });
+
```

Comment:
Ideally we should change the application audience from this interface it self. But for right now I think its okay.

Reply:
Ack